src/render_gl1.c

```diff
@@ -223,24 +229,17 @@ static boolean gl1_set_video_mode(struct graphics_data *graphics,
         return false;
       }
 
-      if(strstr(extensions, "GL_ARB_texture_non_power_of_two"))
```

Comment:
One more request. Since you're removing the NPOT check can you update the comment above this section (probably just remove lines 206 and 207)?

Reply:
Fixed.